packages/components/addon/components/hds/dropdown/list-item.hbs

```diff
@@ -14,28 +14,46 @@
 {{else if (eq this.item "interactive")}}
   <li class={{this.classNames}}>
     {{#if @route}}
-      {{! // TODO consider approach }}
-      <LinkTo @route={{@route}} class="is-{{@state}}">
+      <LinkTo
+        class="{{if @state (concat 'is-' @state)}}"
+        @current-when={{@current-when}}
+        @models={{hds-link-to-models @model @models}}
+        @query={{hds-link-to-query @query}}
+        @replace={{@replace}}
+        @route={{@route}}
+        ...attributes
+      >
         {{#if @icon}}
-          <div class="hds-dropdown-list-item__link-icon">
+          <div class="hds-dropdown-list-item__interactive-icon">
```

Comment:
not a big fan of the name `interactive` for this element (maybe `action`?)

Reply:
since "link" is explicitly related to an `<a>` element, and "action" with a `<button>`, we settled on "interactive". If there is another word that is not already tied to one element or the other, but would work generically, we could consider, but let's do that separately.